### PR TITLE
protected apartment index

### PR DESCRIPTION
**Pull request by pkamano**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -7,27 +7,58 @@ import ApartmentShow from './pages/ApartmentShow'
...
+
   render() {
+    const {
+      logged_in,
+      current_user,
+      new_user_route,
+      sign_in_route,
+      sign_out_route
+    } = this.props
+    console.log(this.state.apartments)
```
Comment (elysemontano) [inline app/javascript/components/App.js]: Let's remove this console log

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -7,27 +7,58 @@ import ApartmentShow from './pages/ApartmentShow'
...
-          </Switch>
-        </Router>
+      <Router>
+      <Header {...this.props} />
+      <Switch>
+        <Route exact path="/" component={Home} />
+        <Route path="/mylistings" render={(props) => {
+          let myListings = this.state.apartments.filter(apartment => apartment.user_id === current_user.id)
+          return(
+        <ProtectedApartmentIndex apartments={myListings} />)}} />
```
Comment (elysemontano) [inline app/javascript/components/App.js]: Check indentation

File: app/javascript/components/pages/ApartmentIndex.js [hunk 3]
```diff
@@ -1,9 +1,20 @@
...
+  CardTitle, CardSubtitle, Button, Col, Row } from 'reactstrap';
 
 export default class ApartmentIndex extends Component {
   render() {
+    const {
+      logged_in,
+      current_user
+    } = this.props
+    console.log("logged_in:", logged_in)
+    console.log("current_user:", current_user)
```
Comment (elysemontano) [inline app/javascript/components/pages/ApartmentIndex.js]: Let's remove these console logs

File: app/javascript/components/pages/ProtectedApartmentIndex.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+class ProtectedApartmentIndex extends Component {
+  render() {
+    const {
+      logged_in,
+      current_user,
+    } = this.props
+    console.log("logged_in:", logged_in)
+    console.log("current_user:", current_user)
+    console.log(this.props.apartments)
```
Comment (elysemontano) [inline app/javascript/components/pages/ProtectedApartmentIndex.js]: Let's remove these console logs

**Review by elysemontano — CHANGES_REQUESTED** [review]
A few things to clean up.

File: app/javascript/components/pages/ApartmentIndex.js (hunk 3 above)
Comment (pkamano) [reply app/javascript/components/pages/ApartmentIndex.js]: So this was an accident and I don't want to mess up Alex's hard work so I just copied what he had in main for this file and replaced what I had.

**Review by elysemontano — CHANGES_REQUESTED** [review]
I still see some console logs in here.  Please remove them as we want to keep the console clean. (Leave the ones on the Header) all the others can go.

**Review by elysemontano — APPROVED** [review]

**Pull request merged**